host-bin/startxiwi

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh -e
+# Copyright (c) 2015 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -e
+
+APPLICATION="${0##*/}"
+OPTS_ENTER=''
+OPTS_XIWI=''
+
+USAGE="$APPLICATION [options] chroot_app [parameters]
+
+Wraps enter-chroot to launch a window or tab in Chromium OS for any graphical application.
+Applications launched in this way show in independent windows or tabs.
+
+By default, it will use the primary user on the first xiwi-enabled chroot found and launch
+the chroot_app in a window.
+
+    -b          Fork and run the specified command silently in the background.
```

Comment:
Sorry I wasn't clear.  The options derived from enter-chroot can be pulled in via a call to enter-chroot, since that doesn't require root.  The options derived from xiwi are the ones that need to be replicated here.  The result should be presented like it is here.